spec.yml

```
@@ -378,10 +378,160 @@ paths:
                     value: "invalid"
       tags:
         - key-usage
+  /1.1/get-stats:
+    post:
```

```suggestion
    get:
```

I think this is a GET endpoint.

---

The parameters need to be sent as POST parameters: https://github.a8c.com/Akismet/akismet/blob/trunk/rest/1.1/get-stats.php

That endpoint has been around for a long time - 2014!

When I implemented the `/1.2/get-key-stats` endpoint, I decided to keep it very similar to the `/1.1/get-stats` endpoint, so it also requires POST parameters.

It does seem inappropriate to set these as POST endpoints. I could update both endpoints to accept the parameters via GET or POST and then use GET in the specification.

---

Wow - it's vintage! I had no idea. Keeping the new endpoint similar in style makes a lot of sense.

> I could update both endpoints to accept the parameters via GET or POST and then use GET in the specification.

Sounds great 👍 I think that'd definitely be worthwhile before we publish the endpoints to the spec.